doc/widgets/saveimages.md

```diff
@@ -0,0 +1,45 @@
+Save Images
+===========
+
+Save images in the directory structure.
+
+**Inputs**
+
+- Data: images to save.
+
+**Save Images** is a simple widget that saves images sent to its input. Images will be saved as separate files in their own directory.
```

Comment:
I would just add here something like: `When a class is present in the data images will be saved in subdirectories based on the class variable`

Reply:
Otherwise, it looks good. Thank you.